wiki/en/en-Contribution.md

```diff
@@ -8,18 +8,31 @@ permalink: "/wiki/Contribution"
 # Contributing to the Jamulus Project
 
 ## Jamulus Software
-* If you want to contribute code, post your idea on [the developer forum](https://sourceforge.net/p/llcon/discussion/developerforum/) first so that it can be discussed before making a pull request (be sure to read main dev's [Volker's manifesto for contributions](https://github.com/corrados/jamulus/issues/596))
 
-See the [Jamulus GitHub repo](https://github.com/corrados/jamulus) for further information.
+If you want to contribute code or have a feature request, [open an issue on GitHub](https://github.com/jamulussoftware/jamulus/issues/) (preferred) or [post your idea on the forum](https://github.com/jamulussoftware/jamulus/discussions) if you don't have a GitHub account. This allows us to discuss the changes before e.g. making a pull request (be sure to read main dev's [Volker's manifesto for contributions on GitHub](https://github.com/jamulussoftware/jamulus/discussions/915) and the [CONTRIBUTING file](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md))
```

Comment:
@gilgongo this seems to be logical error ;-). If you don't have a GitHub account but want to post an idea on GitHub discussions, this will not work ;-).